pr_agent/algo/utils.py

```diff
@@ -247,7 +247,8 @@ def update_settings_from_args(args: List[str]) -> List[str]:
                 arg = arg.strip('-').strip()
                 vals = arg.split('=', 1)
                 if len(vals) != 2:
-                    logging.error(f'Invalid argument format: {arg}')
+                    if len(vals) > 2: # --extended is a valid argument
+                        logging.error(f'Invalid argument format: {arg}')
```

Comment:
**Suggestion:** The error message in the `update_settings_from_args` function should be more specific to indicate that the argument format is invalid because it contains more than one '=' character.
```suggestion
                    if len(vals) > 2: # --extended is a valid argument
                        logging.error(f'Invalid argument format: {arg}. Argument should not contain more than one  '= ' character.')
```